src/preload.js

```diff
@@ -77,6 +80,28 @@ const elements = {
     return "unknown artist(s)";
   },
 
+  getAlbumName: function () {
+    //If listening to an album, get its name from the header title
+    if(window.location.href.includes('/album/')) {
+      const albumName = window.document.querySelector(this.album_header_title);
+      if(albumName) {
```

Comment:
I take it that the url either includes album or playlist? Never both?
If so, please wrap the second if in an "elseif"